### PR TITLE
Cleaner code generation for MMS source term

### PR DESCRIPTION
**Pull request by tkphd**
Replaces SymPy code for deriving the MMS source term to more closely follow @stvdwtt's derivation, producing more apparent agreement and tidier expressions.

### CODE REVIEW

**Comment by pfhub** [comment]
@tkphd, the new version of the PFHub website is available at https://random-cat-1504.surge.sh

File: benchmarks/benchmark7.ipynb [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "application/javascript": [
```
Comment (wd15) [inline benchmarks/benchmark7.ipynb:10]: <pre class="ql-syntax" spellcheck="false">Line #2.    print("Compare generated expression for S:")
</pre><p>Why not just use the markdown cells for the text? Then the font won't be weird.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/usnistgov/pfhub/pull/1504/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Comment by wd15** [comment]
I can't fix this as my environments don't seem to have sympy

**Comment by wd15** [comment]
Also, https://random-cat-1504.surge.sh/benchmarks/benchmark7.ipynb/#Feedback seems to have broken markdown for the links. I think you just need to reexcute the cell and rebuild the html. I can do that if you sort the issue above.

**Comment by pfhub** [comment]
@tkphd, the new version of the PFHub website is available at https://random-cat-1504.surge.sh

**Comment by pfhub** [comment]
@tkphd, the new version of the PFHub website is available at https://random-cat-1504.surge.sh

**Comment by pfhub** [comment]
@tkphd, the new version of the PFHub website is available at https://random-cat-1504.surge.sh

File: benchmarks/benchmark7.ipynb (hunk 1 above)
Comment (tkphd) [reply benchmarks/benchmark7.ipynb:10]: Sorted in 88099cc

**Comment by tkphd** [comment]
The links are still broken, but the math is better. Thanks!

**Comment by pfhub** [comment]
@tkphd, the new version of the PFHub website is available at https://random-cat-1504.surge.sh

**Pull request merged**